src/app/view/components/dot-message-display/model/dot-message-severity.model.ts

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
I think we should put all this models related to this feature in the same file... what do you think @alfredo-dotcms @hmoreras ?

Reply:
didn't we agreed on adding a `index.js` and include them?

Reply:
There are places where we use the same file for all models in the module like `dot-page-selector.models.ts` in separate files looks like more "clean" but to many files with a couple of lines. Keep all in one file will avoid the index.js idea.

Reply:
One of the benefits is you don't have to import files to use the types.